csharp-basics/exercises/Arithmetic/Exercise10/Geometry.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculateArea
+{
+    public class Geometry
+    {
+        public static double areaOfCircle(decimal radius)
+        {
+            if (radius < 0)
+            {
+                throw new ArgumentException("Parameter cannot be negative", nameof(radius));
```

Comment:
un ja ir 0?

Reply:
Pag, vai tad rādiuss nedrīkst arī būt 0? Jo tas skaitās kā nenegatīvs skaitlis un tādēļ kļūdu skatījos tiem skaitļiem, kas ir mazāk par 0 (pašu neieskaitot), vnk uzdevumā bija minēts, ka nav atļauti tikai negatīvie skaitļi - jāizvada kļūda to gadījumā!

Reply:
Nekas, sazinājos ar Edgaru un visu samainīju